.env.prod.example

```
@@ -0,0 +1,27 @@
+# Production Environment Configuration
+# Copy this to .env.prod for production deployment
+# 
+# NOTE: Most configuration is now handled by app_config.nix and conf/db.yaml
+# This file contains only environment-specific overrides
+
+# Application Mode
+ENDO_API_MODE=production
+
+# Django Settings (Required)
+DJANGO_DEBUG=False
+DJANGO_ALLOWED_HOSTS="your-domain.com,localhost"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)